apps/condo/domains/ticket/schema/TicketComment.test.js

```diff
@@ -879,7 +879,7 @@ describe('TicketComment', () => {
         })
 
         describe('update', () => {
-            it('can update own comment', async () => {
+            it('cannot update own comment', async () => {
```

Comment:
Why do we restrict updating users' own comments? Is it a business requirement?

Reply:
Yes, we decided that the resident will work with his tickets in the same way as with tickets for him from crm. Resident will not be able to edit their tickets or tickets from crm